protected/gii/CaseSearchParameter/CaseSearchParameterCode.php

```diff
@@ -12,12 +12,13 @@ class CaseSearchParameterCode extends CCodeModel
     public $alias;
     public $name;
     public $attributeList;
-    public $searchProviders;
+    public $searchProviders = 'DBProvider'; // default to DBProvider
+    public $path = 'application.modules.OECaseSearch';
 
     public function rules()
     {
         return array_merge(parent::rules(), array(
-            array('className, name, alias, searchProviders', 'required'),
+            array('className, name, alias, searchProviders, path', 'required'),
             array('className, alias', 'match', 'pattern' => '/^\w+$/'),
             array('attributeList, searchProviders', 'match', 'pattern' => '/^[\w,]+$/'),
```

Comment:
That was pretty much copy/pasted from the sample code on the Yii framework site. I only added the comma to allow comma separators in the text.

I'm pondering changing the delimiter to newlines in a future refactor.